bin/install

```diff
@@ -11,111 +11,132 @@ install_elixir() {
     local tmp_download_dir=$TMPDIR
   fi
 
+  # running install commands in a subshell
```

Comment:
I was confused by this at first until I noticed the parentheses inside the if statement. Maybe may this plural? `running install commands in subshells`.

Reply:
Taking a second look, I noticed there's actually no need for both to be running in sub-shells. There's only a small moment where this should happen when installing a reference. See the change in 816ac89 and let me know what you think.